Modules/Core/ImageFunction/include/itkInterpolateImageFunction.h

```diff
@@ -127,12 +129,14 @@ class ITK_TEMPLATE_EXPORT InterpolateImageFunction:
     return ( static_cast< RealType >( this->GetInputImage()->GetPixel(index) ) );
   }
 
+#if !defined(ITKV4_COMPATIBILITY)
   /** Get the radius required for interpolation.
    *
    * This defines the number of surrounding pixels required to interpolate at
    * a given point.
    */
   virtual SizeType GetRadius() const = 0;
```

Comment:
I suggest if compatibility in enabled the the implementation then change this from an abstract implementation to an implementation which returns the largest possible region. This would retain the streaming feature when compatibility is enabled, and still not require derived interpolators to have it implemented, as it would default to a value which disables streaming.

Reply:
This will silently defeat the purpose of the feature.

Reply:
This patch completely removed the feature. And Brad's proposal leaves the feature implemented for built-in classes, but does not force its implementation for external classes when `ITKV4_COMPATIBILITY` is defined. Which is a good path from V4 to V5.